apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -113,6 +114,7 @@ function WpcomBlockEditorNavSidebar() {
 	const handleClose = ( e: React.MouseEvent ) => {
 		if ( hasAction( 'a8c.wpcom-block-editor.closeEditor' ) ) {
 			e.preventDefault();
+			recordTracksEvent( 'calypso_editor_sidebar_close' );
```

Comment:
```suggestion
			recordTracksEvent( 'calypso_editor_sidebar_close_editor' );
```

"close editor" might help distinguish it from closing the sidebar. Or maybe it should be `_editor_close` since we often put the verb at the end in track event names.

Reply:
Yes, I see what you mean.  'calypso_editor_sidebar_editor_close' follows the pattern, and then intention.  Probably confusing regardless. :). But I suppose that is where https://mc.a8c.com/tracks/events/register/ comes in, to add descriptions & clairty.